### PR TITLE
Underscore fixes

### PR DESCRIPTION
**Pull request by bluong**
Some variables were underscored incorrectly and thus are fixed here

### CODE REVIEW

File: src/core/renderController.ts [hunk 1]
```diff
@@ -18,10 +18,10 @@ module Plottable {
...
    */
   export module RenderController {
-    var _componentsNeedingRender = new Utils.Set<Component>();
-    var _componentsNeedingComputeLayout = new Utils.Set<Component>();
-    var _animationRequested = false;
-    var _isCurrentlyFlushing = false;
+    var componentsNeedingRender = new Utils.Set<Component>();
+    var componentsNeedingComputeLayout = new Utils.Set<Component>();
+    var animationRequested = false;
+    var isCurrentlyFlushing = false;
```
Comment (aicioara) [inline src/core/renderController.ts]: Why this change? These are all private variables.

Comment (bluong) [inline src/core/renderController.ts]: They are not exposed to the user.  There's no way for them to access these vars.

Comment (aicioara) [inline src/core/renderController.ts]: What is the issue with the underscore than?

Comment (bluong) [inline src/core/renderController.ts]: Having the underscore convention here is similar to having underscores for our local variables within function definitions.  The intention of the underscore was so if a user were to call like object._someFunction, they would be signaled that the function usage as at your own risk.  However, since these variables are not even exported, there is no RenderController._componentsNeedingRender being shown to the user.

Comment (aicioara) [inline src/core/renderController.ts]: Okay then

Comment (aicioara) [inline src/core/renderController.ts]: I think it is more consistent with the rest of the rest of our codebase. Underscored always means not exported, why are these more special?

Comment (jtlan) [inline src/core/renderController.ts]: Agree with @aicioara here -- while it's not necessary to underscore them, it will result in less confusion.

**Comment by aicioara** [comment]
NOQE because it is just private method rename
